LINK/usr/bin/miq_postgres_upgrade.sh

```diff
@@ -99,9 +104,41 @@ mount -a
 # init the new cluster
 su - postgres -c "initdb -D ${NEW_PGSQL_DIR}/data-new"
 
+sed -i.bak -e "s/^#*shared_preload_libraries.*/shared_preload_libraries = 'pglogical'/" ${NEW_PGSQL_DIR}/data-new/postgresql.conf
```

Comment:
It wont be.
We are editing the base `postgresql.conf` file which is generated from the `initdb` in the previous line.

On that node we can probably just append this like we do with the other two below because this is also commented by default.